fasttrips/Examples/demand/psrc_1_1/trip_list.txt

```diff
@@ -0,0 +1,99 @@
+person_id,person_trip_id,o_taz,d_taz,mode,purpose,departure_time,arrival_time,timePeriod,time_target,vot
```

Comment:
Suggest we have a simplified composition and number of trips if we are going to commit this to the Examples directory.  This will greatly simplify debugging, particularly of pathweights_ft.  Perhaps we just have work and social trips?

Reply:
Check.